### PR TITLE
Enables Flake8 E501

### PR DESCRIPTION
**Pull request by nicomiguelino**
#### Description

* See https://www.flake8rules.com/rules/E501.html for details.

### CODE REVIEW

File: lib/assets_helper.py [hunk 1]
```diff
@@ -118,7 +120,10 @@
     if 'is_active' in asset:
         asset.pop('is_active')
     with db.commit(conn) as c:
-        c.execute(queries.update(list(asset.keys())), list(asset.values()) + [asset_id])
+        c.execute(
+            queries.update(list(asset.keys())),
```
[the remark this answers is not in the document]
Comment (vpetersson) [reply lib/assets_helper.py:124]: This is a good catch. We have a possible SQL injection vector here.

**Review by vpetersson — APPROVED** [review]
Should probably fix the sql injection issue.

File: lib/assets_helper.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (nicomiguelino) [reply lib/assets_helper.py:124]: @vpetersson, thanks for pointing this out. I'll either do this here or on a separate PR.

[the remark this answers is not in the document]
Comment (vpetersson) [reply lib/assets_helper.py:124]: This wasn't introduced in the PR. I'm thinking it makes more sense to solve this in the Django rewrite.

[the remark this answers is not in the document]
Comment (nicomiguelino) [reply lib/assets_helper.py:124]: Yes, it makes more sense to do it during the data migration, where those raw SQL statements will be converted to use Django's QuerySet API.

**Pull request merged**